whisper.py

```diff
@@ -1,74 +1,49 @@
-from models import (
-    WhisperASRInput,
-    WhisperASROutput,
-    OutputType,
-)
-import logging
-import time
+# import ast
 import json
-import ast
-from dane.config import cfg
-from dane.provenance import Provenance
-from io_util import (
-    get_base_output_dir,
-    get_output_file_path,
-    check_model_availability,
-    check_pretrained_model_availability,
-)
+import logging
+import os
+
 import faster_whisper
+from config import (
+    model_base_dir,
+    w_beam_size,
+    w_best_of,
+    w_device,
+    # w_model, TODO handle model download later
+    # w_temperature,
```

Comment:
Were there issues with this parameter?

Reply:
ah yes the temperature did indeed give some problems. I disabled it temporarily, but forgot to try it again.

I was testing cpu only earlier when it failed. Maybe using cuda it works? (haven't really dived into the whisper side of things)

Reply:
I only had issues with it when it came to parsing the variable from the env file. But I will have another look when I test with the GPU